docs/guide/deployment/oauth-server.md

```diff
@@ -0,0 +1,134 @@
+# OAuth Server
+OAuth Server acts as an OAuth provider for mock testing to authorizes users and devices interacting with the plgd cloud.
```

Comment:
Keeps it simple, without repetitions. 

Mocked OAuth2.0 Server used for automated tests and [bundle container](...) (keep dots, we will update later).